### ISSUE TITLE
suggestion: Include additional details in the Wiki about systems in the game to help players

### ISSUE THREAD

**Issue by DaniRainbow** [danenders/pokeclassic#135]
I think it would be helpful to have the Wiki updated with information about systems to better help the players.

For example, if Pokemon Classic uses AI from a particular gen or it uses a battle system made by fans (like the framework for Radical Red) it would be nice to have a link to that gen or fan system so players can read about how the AI is different than the vanilla Fire Red/ Leaf Green. 

It looks like there may be something similar to this in the works on the Wiki page, but anything that is a quality of life change or uses another project/system could be notated to improve the user experience.

This rom hack is so well done and this could take it another step greater as far as player support is concerned.

**Comment by danenders** [comment]
Yeah 😓 , the wiki started out as a place for me to keep notes when i was working on this more actively last summer and then i fell behind. I'm bad at keeping records but I agree we should update it.

**Comment by DaniRainbow** [comment]
If you want help and I can provide it, let me know.

**Comment by MrMazzone** [comment]
I'll jump in too and help document/update wiki. Let me take a look.
